content/docs/09.administrator-guide/02.deployment/07.google-vm.md

```diff
@@ -0,0 +1,197 @@
+
+![vm creation 2](/docs/administrator-guide/deployment/gcp-vm/vm_setup2.png)
+
+![change_boot_disk_image](/docs/administrator-guide/deployment/gcp-vm/vm_setup3.png)
+
+![vm_creation_3](/docs/administrator-guide/deployment/gcp-vm/vm_setup4.png)
+
+You can now click on "Create" and wait a few seconds for the VM instance to be up and running.
+
+Then, click on the `SSH` button on the right of the VM instance details to SSH into the VM instance terminal. Click on the `Authorize` button in the pop-up to authorize the SSH connection into the VM instance.
+
+![ssh_into_vm](/docs/administrator-guide/deployment/gcp-vm/ssh_into_vm.png)
+
+Kestra can be run directly from the .jar binary or thanks to Docker. We’ll use docker here for quicker setup:
+1. Install Docker on the EC2 instance. [You can find the last updated instruction on Docker website](https://docs.docker.com/engine/install/ubuntu/).
+2. [Install docker-compose](https://docs.docker.com/compose/install/).
+
+To check your installation run `docker version` and `docker-compose version`. You're now ready to download and launch Kestra server.
+
+
+## Step 2: Download and Run Kestra
+
+Download the official Docker-Compose file: `curl -o docker-compose.yml https://raw.githubusercontent.com/kestra-io/kestra/develop/docker-compose.yml`
+
+Use editor such as Vim to modify the `docker-compose.yml` and set basic authentication to true, and configure your basic authentication credentials to secure your Kestra instance.
+
+```yaml
+kestra:
+  server:
+    basic-auth:
+      enabled: true
+      username: admin@kestra.io # it must be a valid email address
+      password: kestra
+```
+
+Then use the following command to start Kestra server: `sudo docker-compose up -d`
+
+
+## Step 3: Allow external traffic
+
+Kestra is now running and the Kestra server exposes traffic on the 8080 port. To connect through your web browser, we need to allow the traffic from port 8080 to the VM.
+
+1. Go to the Compute Engine console, and click on the three dots to the utmost right on the VM instance row. From the dropdown menu, click on `View Network Details`.
+2. On the Network Interface Details page, click on `Firewall` from the left side navigation menu.
+3. On the Firewall Policies page, click on `Create Firewall Rule` button at the top of the page.
+4. Give an appropriate name like `kestra-ui-port`, and an appropriate description to the Firewall Rule.
+5. In the targets, select `All instance in the network`, select `IPv4 ranges` as the Source filter, and provide appropriate IPv4 address or range in the Source IPv4 ranges.
+   If you want to only allow traffic coming from your IP address, put your own IP. If you want to make it open to the entire Internet, put the value `0.0.0.0/0`.
+6. Under Protocols and ports section, in the Specified protocols and ports, select `TCP` and enter `8080` in the Ports.
+7. Click on `Create` at the bottom of the page.
```

Comment:
```suggestion
Download the official Docker-Compose file: `curl -o docker-compose.yml https://raw.githubusercontent.com/kestra-io/kestra/develop/docker-compose.yml`

Use an editor such as Vim to modify the `docker-compose.yml`, set basic authentication to true, and configure your basic authentication credentials to secure your Kestra instance.

```yaml
kestra:
  server:
    basic-auth:
      enabled: true
      username: admin@kestra.io # it must be a valid email address
      password: kestra
```

Then, use the following command to start the Kestra server: `sudo docker-compose up -d`


## Step 3: Allow external traffic

Kestra is now running, and the Kestra server exposes traffic on the 8080 port. To connect through your web browser, we need to allow the traffic from port 8080 to the VM.

1. Go to the Compute Engine console and click on the three dots to the utmost right on the VM instance row. From the dropdown menu, click on `View Network Details`.
2. On the Network Interface Details page, click on `Firewall` from the left-side navigation menu.
3. On the Firewall Policies page, click on the `Create Firewall Rule` button at the top of the page.
4. Give an appropriate name, such as `kestra-ui-port`, and an appropriate description for the Firewall Rule.
5. In the targets, select `All instance in the network`, select `IPv4 ranges` as the Source filter, and provide the appropriate IPv4 address or range in the Source IPv4 ranges.
   If you want to only allow traffic coming from your IP address, put your own IP. If you want to make it open to the entire Internet, put the value `0.0.0.0/0`.
6. Under the Protocols and ports section, in the Specified protocols and ports, select `TCP` and enter `8080` in the Ports.
7. Click on `Create` at the bottom of the page.
```